auth/README.md

```diff
@@ -0,0 +1,13 @@
+# How to deploy
+1. Clone bahis-oidc in the same level as bahis-infra
+```bash
+git clone https://github.com/road86/bahis-oidc
```

Comment:
Why not just use submodules like dash?

Reply:
Then all the applications will be submodules of bahis infra.

Reply:
I think that's probably the way to go. Are you happy doing that?

Reply:
sure. i am doing